src/client/common/cy/revisions/rearrangeGraph.js

```diff
@@ -0,0 +1,30 @@
+const _ = require('lodash');
+const apiCaller = require('../../../services/apiCaller/');
+
+/* rearrangeGraph(layouts, positions, cy, headless, options)
+Apply a human created layout and submit the changes to the server
+Returns a PNG of the modified graph
+Note : -Options Parameter is optional
+       -Layouts is a collection of layouts
+       -Positions is a collection of position objects 
+*/
+function rearrangeGraph(positions, cy, options) {
+  Object.keys(positions).forEach(function (key) {
+    let node = cy.getElementById(key);
```

Comment:
This code is confusing.  Why is positions a key for a cytoscape node?

Reply:
@d2fong  `positions` is actually an object consisting of node positions indexed by node id,  i.e {nodeKey : nodePosition}, so thats why the key is being used to reference a cytoscape node. 

I have renamed the variable to make it slightly more clear.